ramls/examples/tags/tags_get_200_response.json

```diff
@@ -0,0 +1,12 @@
+{
+  "data": [
```

Comment:
Do we need to change this response to suit the new format?

Reply:
updated

Reply:
👍